README.rst

```diff
@@ -216,3 +217,16 @@ Documentation for other versions
 * `v0.1.1 <https://www.pygmt.org/v0.1.1>`__
 * `v0.1.0 <https://www.pygmt.org/v0.1.0>`__
 * `v0.0.1a0 <https://www.pygmt.org/0.0.1a0>`__
+
+Compatibility with Python and GMT versions
+------------------------------------------
+
+======= ========== =========
+PyGMT   GMT        Python
+======= ========== =========
+0.2.1   >=6.1.1    3.6 - 3.9
+0.2.0   >=6.1.1    3.6 - 3.8
+0.1.2   >=6.0.0    3.6 - 3.8
+0.1.1   >=6.0.0    3.6 - 3.8
+0.1.0   >=6.0.0    3.6 - 3.8
```

Comment:
Perhaps we can merge the three versions into one entry `0.1.x`?

Reply:
I just ran PyGMT v0.1.0 with GMT master branch. Although some tests fail, the core function of PyGMT still works, so I'm hesitant to say that 0.1.0 required GMT>=6.0.0, <=6.1.0

Reply:
Yep, merging them into 0.1.x sounds good. And agree that we can drop the <6.1.0 part too.

Reply:
Done.